src/dotnet/Users.Contracts/Backend/IRecentsBackend.cs

```diff
@@ -0,0 +1,21 @@
+namespace ActualChat.Users;
+
+public interface IRecentsBackend
+{
+    [ComputeMethod]
+    Task<ImmutableHashSet<string>> List(
+        string shardKey,
+        RecentScope scope,
+        int limit,
+        CancellationToken cancellationToken);
+    [CommandHandler]
```

Comment:
Maybe makes sense to add an extra empty line between compute methods and command handlers.

Reply:
done